### PR TITLE
working-groups: update wg-key-visualizer

### PR DESCRIPTION
**Pull request by HunDunDM**
update key visualizer working group meeting notes link and agenda link

### CODE REVIEW

File: working-groups/wg-key-visualizer.md [hunk 1]
```diff
@@ -4,13 +4,16 @@
...
+[Proposal](https://github.com/pingcap/community/issues/88)
+
+[RFC](https://github.com/tikv/rfcs/pull/36)
 
 ## Work and Meeting
 
 * Regular Working Group Meeting: 4pm on every Thursday
 * Meeting Zoom: [https://pingcap.zoom.com.cn/j/669899354](https://pingcap.zoom.com.cn/j/669899354)
-* [Work Status & Materials](https://docs.google.com/document/d/1LdvS0oWo79eTINHCZAdI0z0vKGP2xiCLnHYKRHBwfh8)
+* Agenda: [https://github.com/orgs/pingcap/projects/4](https://github.com/orgs/pingcap/projects/4)
```
Comment (winkyao) [inline working-groups/wg-key-visualizer.md]:
```suggestion
* Progress: [https://github.com/orgs/pingcap/projects/4](https://github.com/orgs/pingcap/projects/4)
```

File: working-groups/wg-key-visualizer.md [hunk 2]
```diff
@@ -4,13 +4,16 @@
 
 The Key Visualizer Working Group aims to implement a visual component to show key heatmap of the TiKV cluster, which is useful for troubleshooting inefficient application usage patterns.
 
-[RFC](https://github.com/tikv/rfcs/pull/367)
+[Proposal](https://github.com/pingcap/community/issues/88)
+
+[RFC](https://github.com/tikv/rfcs/pull/36)
```
Comment (zz-jason) [inline working-groups/wg-key-visualizer.md]: please use this format:

```
* Proposal: [title](url)
* RFC: [title](url)
```

File: working-groups/wg-key-visualizer.md [hunk 3]
```diff
@@ -4,13 +4,16 @@
 
 The Key Visualizer Working Group aims to implement a visual component to show key heatmap of the TiKV cluster, which is useful for troubleshooting inefficient application usage patterns.
```
Comment (dcalvin) [inline working-groups/wg-key-visualizer.md]:
```suggestion
The Key Visualizer Working Group aims to implement a visual component to show key heatmap of the TiKV cluster, which is useful for troubleshooting TiKV performance issues and identify inefficient application usage patterns.
```

**Comment by gingerkidney** [comment]
LGTM

**Pull request merged**